### ISSUE TITLE
Mob Creation

### ISSUE THREAD

**Issue by WhiskeyTangoFox1** [Urothis/nwn-howardslotr#3]
It is my speculation that a DM spawning an enemy (player interaction) will spawn an enemy that drops "ALL" of his equipment. It is my recommendation that we limit what certain enemies drop so that we can better plan player rewards, item control, etc. etc.

**Comment by Yazkir** [comment]
The NPC should still only drop their 'droppable' loot when killed. A simple but somewhat time consuming solution to this problem could be duplicating NPC's and then having a DM spawned NPC and a normal spawned NPC. Which would allow for one of their inventories to only contain undroppable items and the other to be a standard lootable NPC. With the number of NPCs in this module I am not sure if that is a viable solution. 

(Not speaking on behalf or Urothis or the devs, just commenting in general on this specific issue.)

I will look into seeing if there is a simpler scripting solution for this.

**Comment by winternite** [comment]
as the only one who is running around as DM these days, when I have the few times spawned monsters they had the normal loot table.

**Comment by Yazkir** [comment]
Yeah, the loot table doesn't change. It should drop the same items it would if it were to normally spawn.
